src/Client/Web/Bit.Client.Web.BlazorUI/Components/TextField/BitTextField.Fluent.scss

```diff
@@ -57,6 +57,16 @@
     font-size: rem(14px);
     font-weight: 400;
     font-family: "Segoe UI", "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;
+
+    .bit-icon {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: rem(32px);
+        height: 100%;
+        font-size: rem(16);
+        color: $Grey160;
```

Comment:
This PR is about the `BitMessageBar` component.
Why do we have some changes in this file?
@RezaKargar

Reply:
The styles are for **bit-icon** inside the **BitTextField** but it was not wrapped inside the corresponding css selector so they were effecting the whole bit-icons even the ones inside **MessageBar**

Reply:
> The styles are for **bit-icon** inside the **BitTextField** but it was not wrapped inside the corresponding css selector so they were effecting the whole bit-icons even the ones inside **MessageBar**

I got it...Thank you :)